lib/gateway/Shard.js

```diff
@@ -1120,7 +1120,8 @@ class Shard extends EventEmitter {
                     oldMember = {
                         roles: member.roles,
                         nick: member.nick,
-                        premiumSince: member.premiumSince
+                        premiumSince: member.premiumSince,
+                        pending: member.pending
```

Comment:
Would it be useful to have a `guildMemberVerified` (or similar) event here?

Reply:
as mentionned in Discord, I don't think it's appropriate to have such event. Doesn't seem to fit in eris